From eab237cab550733a6a46154d3ad5463856479e6f Mon Sep 17 00:00:00 2001
From: Richard Zhao <b20223@freescale.com>
Date: Wed, 10 Sep 2008 13:56:49 +0800
Subject: [PATCH] ENGR00090993 add GPL license statement in unifi glue driver files

Add GPL license statement in:
drivers/mmc/card/unifi_fs/fs_lx.c
drivers/mmc/card/unifi_fs/fs_sdio_api.h

Signed-off-by: Richard Zhao <b20223@freescale.com>
---
 drivers/mmc/card/unifi_fs/fs_lx.c       |   22 +++++++++-------------
 drivers/mmc/card/unifi_fs/fs_sdio_api.h |   10 +++++++++-
 2 files changed, 18 insertions(+), 14 deletions(-)

diff --git a/drivers/mmc/card/unifi_fs/fs_lx.c b/drivers/mmc/card/unifi_fs/fs_lx.c
index 41b9d2b..0f7e173 100644
--- a/drivers/mmc/card/unifi_fs/fs_lx.c
+++ b/drivers/mmc/card/unifi_fs/fs_lx.c
@@ -1,19 +1,8 @@
 /*
- * Freescale SDIO glue modules.
+ * fs_lx.c - Freescale SDIO glue module for UniFi.
  *
  * Copyright (C) 2008 Cambridge Silicon Radio Ltd.
  *
- *  Note:
- *      The Freescale MMC/SDIO driver is a project under development so the
- *      code that interfaces their driver is likely to have changed between
- *      releases. This code is tested only with the SDIO/MMC driver released
- *      by Freescale for the imx31_3stack board using Linux Kernel 2.6.24.
- *
- *      Also, the code in the probe that sets the pull-ups is platform
- *      specific and should really be part of the controller's initialisation.
- *      In the aforesaid Freescale release this code was missing but may
- *      be in place in the future releases.
- *
  *  Important:
  *      This module does not support more than one device driver instances.
  *
@@ -21,7 +10,14 @@
 /*
  * Copyright 2008 Freescale Semiconductor, Inc. All Rights Reserved.
  */
-
+/*
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
 #include <linux/version.h>
 #include <linux/module.h>
 #include <linux/delay.h>
diff --git a/drivers/mmc/card/unifi_fs/fs_sdio_api.h b/drivers/mmc/card/unifi_fs/fs_sdio_api.h
index 1f0c570..ea6ebd7 100644
--- a/drivers/mmc/card/unifi_fs/fs_sdio_api.h
+++ b/drivers/mmc/card/unifi_fs/fs_sdio_api.h
@@ -1,5 +1,5 @@
 /*
- * Freescale SDIO glue module API.
+ *fs_sdio_api.h - Freescale SDIO glue module API for UniFi.
  *
  * Copyright (C) 2008 Cambridge Silicon Radio Ltd.
  *
@@ -7,6 +7,14 @@
 /*
  * Copyright 2008 Freescale Semiconductor, Inc. All Rights Reserved.
  */
+/*
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
 #ifndef _FS_SDIO_API_H
 #define _FS_SDIO_API_H
 
-- 
1.5.4.4

